### DISCUSSION TITLE
More info about cars SoC

### DISCUSSION THREAD

**Discussion by stevoh6** [serkri/SmartEVSE-3#181] [Q&A] [answered]
I found that some car support reporting their SoC. 
https://github.com/serkri/SmartEVSE-3/pull/160

Can you please provide more info, how to test it if my car sopport it? I see that there is mention od ISO modem, what it means? Thx.

**Comment by dingo35** [comment]
You need special hardware and lot of specialist knowledge.

**Reply by stevoh6** [reply]
Could you please advise where to start ? Some good article, tutorial, anything? Thx.

**Comment by dingo35** [comment, marked as answer]
Here is all the expert info:
https://github.com/SmartEVSE/SmartEVSE-3/issues/25

**Reply by stevoh6** [reply]
thx. I overlook this.
